public/js/controllers/game.js

```diff
@@ -1,184 +1,177 @@
 angular.module('mean.system')
 .controller('GameController', ['$scope', 'game', '$timeout', '$location', 'MakeAWishFactsService', '$dialog', function ($scope, game, $timeout, $location, MakeAWishFactsService, $dialog) {
-    $scope.hasPickedCards = false;
-    $scope.winningCardPicked = false;
-    $scope.showTable = false;
-    $scope.modalShown = false;
-    $scope.game = game;
-    $scope.pickedCards = [];
-    var makeAWishFacts = MakeAWishFactsService.getMakeAWishFacts();
-    $scope.makeAWishFact = makeAWishFacts.pop();
-
-    $scope.pickCard = function(card) {
-      if (!$scope.hasPickedCards) {
+  $scope.hasPickedCards = false;
+  $scope.winningCardPicked = false;
+  $scope.showTable = false;
+  $scope.modalShown = false;
+  $scope.game = game;
+  $scope.pickedCards = [];
+  let makeAWishFacts = MakeAWishFactsService.getMakeAWishFacts();
+  $scope.makeAWishFact = makeAWishFacts.pop();
+
+  $scope.pickCard = function (card) {
+    if (!$scope.hasPickedCards) {
         if ($scope.pickedCards.indexOf(card.id) < 0) {
           $scope.pickedCards.push(card.id);
           if (game.curQuestion.numAnswers === 1) {
             $scope.sendPickedCards();
             $scope.hasPickedCards = true;
           } else if (game.curQuestion.numAnswers === 2 &&
             $scope.pickedCards.length === 2) {
-            //delay and send
+            // delay and send
             $scope.hasPickedCards = true;
             $timeout($scope.sendPickedCards, 300);
           }
         } else {
           $scope.pickedCards.pop();
         }
       }
-    };
+  };
 
-    $scope.pointerCursorStyle = function() {
-      if ($scope.isCzar() && $scope.game.state === 'waiting for czar to decide') {
-        return {'cursor': 'pointer'};
-      } else {
-        return {};
+  $scope.pointerCursorStyle = function () {
+    if ($scope.isCzar() && $scope.game.state === 'waiting for czar to decide') {
+        return { cursor: 'pointer' };
```

Comment:
Can you ensure that the indentation remains consistent. You might need to configure your editor to auto fix indentation for you.